docs/setup.rst

```diff
@@ -193,6 +193,11 @@ Docker Method
      container and thus the one of the consumption directory. Furthermore, you
      can change the id of the default user as well using ``USERMAP_UID``.
 
+  ``USE_SSL``
+    If you want Paperless to use SSL for the user interface, set this variable
+    to ``true``. You also need to copy your certificate and key to the ``data``
+    directory, named ``ssl.cert`` and ``ssl.key``.
```

Comment:
Given the discussion in the ticket, we should maybe mention that this setup is not ideal and one should rather use e.g. Nginx for proper setups.

Reply:
I'm not really sure what to write here. If you (or anyone else) have any better ideas, please tell me :)

Reply:
> If you want Paperless to use SSL directly - which is a discouraged setup, try using e.g. Nginx as reverse proxy terminating SSL instead - set this variable […]
Something like that maybe?

Reply:
Have you seen the added sentence in my latest commit? I think it is similar enough.

Reply:
You're right, I didn't see that :)